CMakeLists.txt

```diff
@@ -102,7 +102,7 @@ find_package(Boost COMPONENTS filesystem)
 include_directories(${Boost_INCLUDE_DIRS})
 
 # TODO: utils-urdf should only be required for aikido::perception.
-find_package(DART REQUIRED COMPONENTS optimizer-nlopt utils-urdf CONFIG)
+find_package(DART 6.1.0 REQUIRED COMPONENTS optimizer-nlopt utils-urdf CONFIG)
```

Comment:
I believe the decision of compatibility is determined by the upstream version check script. In particular, the compatibility policy of DART is specified as [_same major version_](https://github.com/dartsim/dart/blob/master/CMakeLists.txt#L363). So this should work for DART 6.1.0 or any greater.